### ISSUE TITLE
Redis image

### ISSUE THREAD

**Issue by patie** [azukiapp/azk#558]
Hi,

your project looks good, how about redis image? I see in example redis, but its no longer in docker hub and github repo.

thanks for explanation

**Comment by gullitmiranda** [comment]
hi @patie, tks.

You can use https://hub.docker.com/r/_/redis/. Which of our example is using an image that does not exist?

**Comment by patie** [comment]
for example in http://azk.io/#azkfile on homepage there is non-existing azukiapp/redis

  database: {
    **image: { docker: "azukiapp/redis" },**
    mounts: { "/data": persistent("data") },
    ports: {
      data: "6379/tcp",
    },
    export_envs: {
      REDIS_URL: "redis://#{net.host}:#{net.port.data}/database",
    },
  }

**Comment by gullitmiranda** [comment]
Sorry, I thought we had already fixed it. I will make the correction now.

The images of docker mostly are compatible with the azk. So far, I have not found any that is not compatible.

**Comment by patie** [comment]
i try it later today, i write here later if its working

**Comment by gullitmiranda** [comment]
ok, I'll wait then answer.

**Comment by dynaum** [comment]
I'm using redis but after azk 0.16.1 I can't start the server.

```
zk: Run system `redis` return: (1), for command: redis-server -c redis-server --bind 0.0.0.0:
azk:  .redis [log] >  1:C 09 Nov 19:08:23.894 # Fatal error, can't open config file '-c'
azk:  .redis [log] >

azk: Look for azk start troubleshooting documentation for more info at: http://bit.ly/azk_start_troubleshooting

azk: Due to the above error azk will stop all instances already running.
```

**Comment by dynaum** [comment]
Using this on Azkfile:

```js
redis: {
  scalable           : false,
  image              : { docker: "library/redis" },
  mounts             : { "/data" : persistent("redis-#{manifest.dir}") },
  export_envs        : {
    REDIS_PORT: "#{net.port[6379]}",
    REDIS_HOST: "#{net.host}",
  },
  http               : { domains: ["#{manifest.dir}-#{system.name}.#{azk.default_domain}"] },
  wait               : { retry: 20, timeout: 2000 },
  ports              : { data: "6379/tcp", },
  command            : "redis-server --bind 0.0.0.0"
}
```

**Comment by dynaum** [comment]
Works removing the command line. :smile:

**Comment by gullitmiranda** [comment]
As we talked at [Gitter](https://gitter.im/azukiapp/azk/pt?at=5640ef72f3818b13297229c6) (in pt-BR), this is a critical bug. I will mark it as bug.

**Comment by gullitmiranda** [comment]
closed by #564
